user-journey.adoc

```diff
@@ -1,62 +1,91 @@
+== Platform launcher workflows
 
-. *`Zip flow`:* Selected mission is downloaded from the web site as a zip archive and extracted to the local filesystem.
-. *`Git flow`:* Selected mission is saved to a GitHub repository/account, and an Istio enabled application is created in a local Openshift instance.
+There are two separate workflows that you can use with the Fabric8 development platform launcher:
+
+* *Continuous Delivery workflow* (Recommended) - Use this workflow if you want to automatically create the application components under your local OpenShift instance and also generate a repository for the booster application code under your GitHub account.
+* *Build and run locally workflow* -  Use this workflow if you want to download the booster application files as a zip archive and extract it to your local file system.  Note that with this workflow you have to follow the instructions in the mission README file to deploy the application.
 
 == Prerequisites
-. At least 6 Gb of free RAM, 4 CPUs
-. *Install Java JDK 8* (necessary only for `Zip flow` and boosters using Fabric8 Maven Plugin)
-.. http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[JDK 8]
-.. http://maven.apache.org/download.cgi[Maven 3.5]
-. *Set up Git SCM / GitHub account*
-.. Install https://git-scm.com/downloads[Git SCM client]
-.. Set up a https://github.com[GitHub account]
-.. Configure a GitHub https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/proc_creating-a-github-personal-access-token.adoc[API access token].
-. *Configure certificates for Red Hat Container Image Registry*
-... RHEL - Certificates are already installed. No action is required.
-... Fedora/CentOS/VM - it is required to have this package installed in order to install Red Hat certificate. This allows the docker daemon to access the Red Hat Container Image Registry
+Before you can install Istio and Fabric8, you should already have met the following prerequisites:
+
+* Fedora/CentOS OR Mac OSX OR Ubuntu with at least 6 Gb of RAM and 4 CPUs.
+* Red Hat Docker Registry with the appropriate configuration (see below for configuration instructions)
+* Know the Ethernet IP of your running docker daemon (see below for instructions for determining this IP)
+
+If you are using the Continuous Integration (GitHub) workflow, there are the following additional prerequisites:
+
+* Install the https://git-scm.com/downloads[Git SCM client].
+* Set up a https://github.com[GitHub account] if you do not already have one.
+* Generate a https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/proc_creating-a-github-personal-access-token.adoc[GitHub personal access token].
+
+////
+To install the {launcher} tool on a {OpenShiftLocal}, you must provide the {launcher} tool with a GitHub personal access token. This enables the {launcher} tool to create booster applications and save them as Git repositories in your GitHub namespace.
```

Comment:
{launcher} tool? What is this doing? Should be same as "Fabric8 Launcher" above, right?